### PR TITLE
ENH add quickstart tutorial with example_config.yml

### PR DESCRIPTION
**Pull request by tbng**
Addresses #80
Note that config for running small toy dataset only works after merging https://github.com/benchopt/benchopt/pull/362

### CODE REVIEW

**Comment by mathurinm** [comment]
@josephsalmon would you add something?

File: example_config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+objective:
+  - Lasso Regression[fit_intercept=False,reg=0.1]
+
+dataset: # dataset to run benchmark on
+  - leukemia
+
+solver: # list of example solvers to do benchmark
+  - celer
+  - cd
+  - sklearn
```
Comment (tomMoral) [inline example_config.yml:10]: is FISTA very slow in this context? Else, I would add it and remove `cd` to increase the diversity.

Comment (tbng) [reply example_config.yml:10]: It's true that cd and sklearn might be overlapping, but I think FISTA is quite slow for a quickstart purpose script, especially if we add leukemia data in?

**Comment by mathurinm** [comment]
I think we need to remove celer as it is slow to install (need to compile cython), and add install instructions. Maybe just CD (numba) vs FISTA ? Or sklearn vs FISTA ?

**Comment by josephsalmon** [comment]
CD and FISTA are the best i think. Standard and no install overhead. Maybe Gram-precomputed versions on top?

**Comment by mathurinm** [comment]
CD has numba overhead, I think sklearn is more reliable to install.

But it's not obvious at first that sklearn uses CD, so I'm split

**Comment by josephsalmon** [comment]
2nd thought: add the l_bfgs_b.py? Seems easy and legite as a simple baseline, wdyt?

**Comment by tomMoral** [comment]
For install, would you have time to make the `benchopt install` command run with `--config`?

I would go with scikit-learn and FISTA. L-BFGS is a more exotic choice.

**Comment by mathurinm** [comment]
I could but I think it would not be as useful as for `benchopt run` (and it complicates the code). Would it be OK to add the `benchopt install` command in the readme before the example instead ?

I also agree L-BFGS is less standard in this setting

**Comment by josephsalmon** [comment]
about L-BFGS it is just that before the prox era, old folks would have transform the pb into a constraint one, and solved it with a black box. I believe it is interesting to have this message somewhere...may be not in the examples then.

**Comment by tomMoral** [comment]
As we don't have to pass the options, just the solver names for `benchopt install`, I agree that it is probably less relevant.
It is just a bit tiring to have to open the `.yml` file to know which solver you should install but I think we can do this later if this becomes an issue.

**Comment by mathurinm** [comment]
So you'd do `benchopt install . --config <the_same_file_as_in_benchmark_run.yml>, but retrieve only solver and dataset from it ? That makes sense in my opinion, and since there are only  2 parameters the code will be less complicated than I thought

**Comment by tomMoral** [comment]
> So you'd do `benchopt install . --config <the_same_file_as_in_benchmark_run.yml>, but retrieve only solver and dataset from it ? That makes sense in my opinion, and since there are only 2 parameters the code will be less complicated than I thought

Yes, I would only take the solver and datasets from the config file.

**Comment by mathurinm** [comment]
It would be better to run everything in an env but this breaks, see https://github.com/benchopt/benchopt/issues/366

**Comment by mathurinm** [comment]
Fixed in benchopt, so merging.

**Pull request merged**